src/Main.jl

```diff
@@ -47,10 +48,10 @@ module MainLoop
 			this.scene = Scene()
 			this.input = Input()
 
-			this.cameraBackgroundColor = [0,0,0]
+			this.cameraBackgroundColor = (0,0,0)
 			this.close = false
-			this.debugTextBoxes = []
-			this.events = []
+			this.debugTextBoxes = UI.TextBoxModule.TextBox[]
+			this.events = [] # what is this for?

```

Comment:
This can be removed I believe. I don't think it's used anywhere and is a leftover of old code